packages/ember-runtime/lib/system/core_object.js

```diff
@@ -191,6 +191,10 @@ function makeCtor() {
       this.init.apply(this, args);
     }
 
+    if (this.__postInitInitialization) {
```

Comment:
actually, can we make this a symbol ? That way no user can use it, and it wont collide with anything else.